source/WindowsAPICodePack/Shell/CommonFileDialogs/CommonFileDialog.cs

```diff
@@ -185,7 +185,7 @@ public ShellContainer DefaultDirectoryShellContainer
         public string DefaultExtension { get; set; }
 
         /// <summary>Default file name.</summary>
-        public string DefaultFileName { get; set; }
+        public string DefaultFileName { get; set; } = string.Empty;
```

Comment:
Changed the default value to `string.Empty` because the `SetFileName` method was throwing a null pointer exception.